lib/spacesuit/session_service.ex

```diff
@@ -40,7 +7,8 @@ defmodule Spacesuit.SessionService do
   require Logger
   use Elixometer
 
-  @behaviour SessionService
+  @callback validate_api_token(String.t()) :: Tuple.t()
```

Comment:
When the _only_ reason for a behaviour is for testing, it's pretty common (and acceptable) to move the callbacks/behaviour definition into the module that implements it in prod.

Reply:
This is super useful because I didn't know about Mox and so I'll definitely check it out. I'm a little lost about how it works ATM moment but I'm sure some reading will clear it up. 

In this case we only had one implementation and a mock, but this module is not that generic and the idea was that people could provide their own implementation of a `SessionService` if they needed to handle it differently. Given that I'd like to support that, but that also nobody else seems to be interested, I'm on the fence about merging.

Reply:
Ah, not a problem. I will update the PR to leave the callback in the SessionService behaviour. That is _technically_ a cleaner implementation and definitely the way to go if you want the ~interface~ behaviour for more than just testing. Mox will work just fine that way, too.